src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -112,7 +112,6 @@ function WorkspaceNewRoomPage(props) {
                 _.filter(PolicyUtils.getActivePolicies(props.policies), (policy) => policy.type !== CONST.POLICY.TYPE.PERSONAL),
                 (policy) => ({
                     label: policy.name,
-                    key: policy.id,
```

Comment:
Any reason for this removal?

Reply:
Yes, it is not used - `keyForList` is taking value from `value` prop so there is no need to pass the same value twice

Reply:
@situchan